rclpy/src/rclpy/_rclpy.c

```diff
@@ -60,6 +67,15 @@ rclpy_create_node(PyObject * Py_UNUSED(self), PyObject * args)
   return pynode;
 }
 
+/// Create a publisher
+/*
+ * \param[in] pynode Capsule pointing to the node to add the publisher to
+ * \param[in] pymsg_type ROS 2 type of the message this publisher will send
```

Comment:
I'd recommend wording this one as "type support structure for the message type associated with the publisher". The doc should clue developers using this API about what they need to pass here (type as well as purpose). It's less import, in my opinion, to convey what this method will do with the parameter, e.g. in this case mentioning that the publisher will use this to send data of this type. If that information is important, I'd recommend specifying it in the body of the doc string rather than on the parameter description.